moped-editor/src/styles/mapboxDrawStylesOverrides.js

```diff
@@ -83,23 +85,23 @@ const mapboxDrawStylesOverrides = [
       ["!=", "mode", "static"],
     ],
     layout: {
-      "line-cap": "square",
+      "line-cap": "round",
     },
     paint: {
-      "line-color": theme.palette.primary.main,
-      "line-width": 4,
+      "line-color": COLORS.orange,
+      "line-width": lineWidthStops,
```

Comment:
Updates in this file are to make sure all lines shown in the map and the map draw tools have identical shape and width.